src/Dfe.PlanTech.Application/Persistence/Models/GetEntitiesOptions.cs

```diff
@@ -0,0 +1,30 @@
+using Dfe.PlanTech.Application.Persistence.Interfaces;
+
+namespace Dfe.PlanTech.Application.Persistence.Models;
+
+public class GetEntitiesOptions : IGetEntitiesOptions
+{
+    public GetEntitiesOptions(int include)
+    {
+        Include = include;
+    }
+
+    public GetEntitiesOptions(int include, IEnumerable<IContentQuery> queries)
+    {
+        Include = include;
+        Queries = queries;
+    }
+
+    public GetEntitiesOptions(IEnumerable<IContentQuery> queries)
+    {
+        Queries = queries;
+    }
+
+    public GetEntitiesOptions()
+    {
+    }
+
+    public IEnumerable<IContentQuery>? Queries { get; init; }
+
+    public int Include { get; init; } = 2;
```

Comment:
Is there only ever going to be to 'things' to include? i.e Header/Content or Title/Content etc.?

Reply:
"Include" here  is the [depth of content references to return](https://www.contentful.com/developers/docs/references/content-delivery-api/#/reference/links) from Contentful's API.

`When you have related content (e.g. entries with links to image assets) it's possible to include both search results and related data in a single request. Using the include parameter, you can specify the number of levels to resolve.

The maximum number of levels is 10. The API will throw a BadRequestError for higher values or values other than an integer. The default number, if the parameter is not specified, is 1. To omit linked items, specify include as 0.`

As we have a `Page` that renders `Content`, and each `Content` reference can have sub-content, we'll potentially need (and _do_ need) greater `include` values than 2 - hence the addition here in options.